src/zkAppWorker.ts

```diff
@@ -98,15 +98,15 @@ const workerFunctions = {
       state.tree.getWitness(state.transitionIndex),
     );
 
-    console.info("creating init proof...");
+    console.info("[zk-states worker] creating init proof...");
```

Comment:
Do we want to keep these console logs in prod version?

Reply:
for now yes, in the future we might consider using a more advanced logging lib